util-images/network/netperfbenchmark/cmd/main.go

```diff
@@ -17,9 +17,15 @@ limitations under the License.
 package main
 
 import (
+	"flag"
+
+	"k8s.io/klog"
 	"k8s.io/perf-tests/util-images/network/netperfbenchmark/pkg/worker"
 )
 
 func main() {
+	klog.InitFlags(flag.CommandLine)
+	flag.Set("logtostderr", "false")
```

Comment:
I would prefer to move it to args, that's how we usually do it.  Example
https://github.com/kubernetes/perf-tests/blob/81c96c34e3c1f11c5fe91744ef7ce7bf44c2fe5c/clusterloader2/pkg/measurement/common/probes/manifests/ping-server-deployment.yaml#L25

Reply:
Modified.